Automattic/vip-go-mu-plugins
# Add homeUrl field to Site Details Sync

## Description

We want to sync the `home_url` option with our Site Details database. 

## Changelog Description

### Changelog Entry Update

Add homeUrl field to Site Details Sync.

It uses the `get_home_url()` [function](https://developer.wordpress.org/reference/functions/get_home_url/) to fetch the data and the Site Details Index to sync

## Checklist


## Steps to Test

1) Use any site
2) Check if the option returns the correct value
